### PR TITLE
[fix]repair the bug which made by myself

### PR DESCRIPTION
**Pull request by YifanZhuZhu**
Just I think,because my bot still can't login.

### CODE REVIEW

File: src/plugins/Core/plugins/_gssearch.py [hunk 1]
```diff
@@ -1,51 +1,49 @@
...
-    if str(message) in urls == False:
-        await gssearch.finish(f"未找到"+message)
-    msg = await gssearch.send(Message("[CQ:image, url="+urls[str(message)]+"]")+f"攻略制作:猫冬\nhttps://www.miyoushe.com/ys/accountCenter/postList?id=74019947")
+    # 稻妻(to do)
 
+    # 须弥(to do)
 
+    if str(message) not in urls:
+        await gssearch.finish(f"未找到" + message)
+    msg = await gssearch.send(Message("[CQ:image, url=" + urls[
```
Comment (This-is-XiaoDeng) [inline src/plugins/Core/plugins/_gssearch.py:45]:
```suggestion
    msg = await gssearch.finish(Message("[CQ:image, url=" + urls[
```

**Review by This-is-XiaoDeng — CHANGES_REQUESTED** [review]
请将文件名修改为`gssearch.py`以启用本插件

**Comment by YifanZhuZhu** [comment]
okl

**Review by This-is-XiaoDeng — COMMENTED** [review]
建议添加一个错误上报

File: src/plugins/Core/plugins/gssearch.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ async def handle_first_receive(event: GroupMessageEvent, bot: bot, message: Mess
 
     if str(message) not in urls:
         await gssearch.finish(f"未找到" + message)
-    msg = await gssearch.finish(Message(f"[CQ:image, url={urls[str(message)]}]\n攻略制作:猫冬 https://www.miyoushe.com/ys/accountCenter/postList?id=74019947"))
+    msg = await gssearch.finish(Message(f"[CQ:image, file={urls[str(message)]}]\n攻略制作:猫冬 https://www.miyoushe.com/ys/accountCenter/postList?id=74019947"))
```
Comment (This-is-XiaoDeng) [inline src/plugins/Core/plugins/gssearch.py:45]:
```suggestion
    msg = await gssearch.finish(Message(f"[CQ:image,file={urls[str(message)]}]\n攻略制作:猫冬 https://www.miyoushe.com/ys/accountCenter/postList?id=74019947"))
```

**Review by This-is-XiaoDeng — CHANGES_REQUESTED** [review]

**Review by This-is-XiaoDeng — APPROVED** [review]
或许可以

**Pull request merged**